doc/index.rst

```diff
@@ -27,7 +28,7 @@ Project `Universum` is a continuous integration framework, containing
 a collection of functions that simplify implementation of the
 automatic build, testing, static analysis and other steps.
 The goal of this project is to provide unified approach for adding continuous integration
-to any project. It currently supports Perforce, Git, Gerrit, Swarm, Jenkins and TeamCity.
+to any project. It currently supports Perforce, Git, Gerrit, Swarm, Jenkins TeamCity and Github Actions.
```

Comment:
1. Comma after "Jenkins" is missing
2. I propose to mention GitHub and GitHub Application. Probably it is worth to rephrase this whole sentence to break it into separate pieces